### ISSUE TITLE
🆕 Software Suggestion | Disk Decipher

### ISSUE THREAD

**Issue by hekkihek** [privacytools/privacytools.io#2346]
## Basic Information

**Name:** Disk Decipher
**Category:** File Encryption
**URL:** https://disk-decipher.app/

## Description

Disk Decipher allows access to encrypted disk images (e.g. TrueCrypt, VeraCrypt or DMG) from your iPhone, iPad and Mac.

## Why I am making the suggestion

It was recommended to me to add this app to your site since there are very few options on Apple mobile devices for using encrypted disk images. This app allows using encryption even on your mobile devices.

I am not going to advertise the app further, this suggestion is just for making people aware that the app exists.

At this moment the app is not open source, which might change in the future.

## My connection with the software

I am the author.

- [x ] I will keep the issue up-to-date if something I have said changes or I remember a connection with the software.

**Comment by youdontneedtoknow22** [comment]
> At this moment the app is not open source, which might change in the future.

I don't think PTIO would ever recommend any closed source app, unless there is absolutely no other alternative.

It's even in the title of that section:

> Discover a variety of open-source software built to protect your privacy and keep your digital data secure.

**Comment by hekkihek** [comment]
AFAIK, there is no open-source alternative for iOS. This listing would help mobile users to access their encrypted data.

**Comment by rusty-snake** [comment]
> AFAIK, there is no open-source alternative for iOS.

Depending on how you categorize iOS but https://www.privacytools.io/operating-systems/#mobile_os

**Comment by hekkihek** [comment]
How would that help users owning an Apple device?

**Comment by AmadeuszL** [comment]
> I don't think PTIO would ever recommend any closed source app, unless there is absolutely no other alternative.
> 
> It's even in the title of that section:
> 
> > Discover a variety of open-source software built to protect your privacy and keep your digital data secure.

Yeah, but [Canary Mail](https://www.privacytools.io/software/email/#iosclients) is still recommended, despite being closed source. 

And to the guy trying to suggest that iOS and apps for it don't belong to PTIO (if I understood him correctly): https://github.com/privacytools/privacytools.io/pull/2075

**Comment by freddy-m** [comment]
> Yeah, but Canary Mail is still recommended, despite being closed source.

It's listed because we needed an iOS email client that had encryption. When we find a proper one it will get de-listed.

**Comment by hekkihek** [comment]
Strange.. what open source software is currently listed for an iOS user to open, say, a VeraCrypt container?

**Comment by rusty-snake** [comment]
According to https://www.veracrypt.fr/en/Android%20%26%20iOS%20Support.html there is only one alternative app for iOS (which I coundn't find on PTIO).

**Comment by hekkihek** [comment]
Precisely. I would suggest listing Disk Decipher until a proper open source alternative is available.

**Comment by freddy-m** [comment]
@privacytools/editorial I'd like your thoughts on this.

**Comment by thunderbird-1990** [comment]
I created a GitHub account specifically to suggest encryption tools on iOS, in response to the recent scandal of Apple.

One of the tools I was going to suggest is Disk Decipher, so I might as well comment here.

To my knowledge, Disk Decipher is one of the only [two applications](https://www.veracrypt.fr/en/Android%20%26%20iOS%20Support.html) that can decrypt VeraCrypt images on iOS. The other application has not been updated since 2019. Neither is open source.

The important bit for me is that Disk Decipher can create VeraCrypt containers locally on an iOS device. Local encryption controlled by users is a rare among iOS applications. Many other apps (e.g., Cryptomator, which I also use) are designed for cloud storage and do not encrypt files locally.

It would be great if the application is open source though. @hekkihek 

Overall, I think it is a good idea to add Disk Decipher to the list. Perhaps a section on the [File Encryption Software](https://www.privacytools.io/software/encryption-tools/) page about options on iOS. There are a lot of "privacy applications" in the App Store but many collect personal data, so I think some guidance should be added to PTIO. @privacytools/editorial

**Comment by hekkihek** [comment]
I hear you. Actually, I have been thinking about open-sourcing the app for
some time. Without making any commitments, I can say that I'm biased
towards making the app open-source.

It's a significant amount of work.

Regards, Richard.

Op zo 15 aug. 2021 om 14:11 schreef thunderbird-1990 <
***@***.***>:
